spec/player_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rspec'
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require 'pry'
+
+describe Player do
+        it 'exists' do
+            card1 = Card.new(:diamond, 'Queen', 12)
+            card2 = Card.new(:spade, '3', 3)
+            card3 = Card.new(:heart, 'Ace', 14)
+            cards = [card1, card2, card3]
+        
+            deck = Deck.new(cards)
+
+            player = Player.new('Clarisa', deck)
+            expect(player).to be_an_instance_of(Player)
+        end
+
+    describe 'has_lost?' do 
+        it 'tells us if the player has lost (true or false)' do
+            card1 = Card.new(:diamond, 'Queen', 12)
+            card2 = Card.new(:spade, '3', 3)
+            card3 = Card.new(:heart, 'Ace', 14)
+            cards = [card1, card2, card3]
+        
+            deck = Deck.new(cards)
+
+            player = Player.new('Clarisa', deck)
+            expect(player.has_lost?).to eq(false)
+        end
+    end
+end
```

Comment:
There are a few tests missing from this spec. We successfully check that a new `Player` instance object can be created and that we can call `has_lost?` on the `Player` object. But We don't check the `Player`'s instance variables like `.name` or `.deck`.

```ruby
expect(player.name).to eq("Clarisa")
expect(player.deck).to eq(deck)
expect(player.deck.cards).to eq(cards)
```